FormalConjectures/Wikipedia/EulerBrick.lean

```diff
@@ -0,0 +1,61 @@
+/-
+Copyright 2025 The Formal Conjectures Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-/
+
+import FormalConjectures.Util.ProblemImports
+
+/-!
+# Open questions on existence of some Euler brick
+
+*References:*
+- [Wikipedia](https://en.wikipedia.org/wiki/Euler_brick)
+- [stackexchange](https://math.stackexchange.com/questions/2264401/euler-bricks-and-the-4th-dimension)
+-/
+
+/--
+**Euler brick** is a rectangular cuboid whose edges and face diagonals all have integer lengths.
+-/
+def IsEulerBrick (a b c : ℕ+) : Prop :=
+  ∃ n m k : ℕ+, a^2 + b^2 = n^2 ∧ a^2 + c^2 = m^2 ∧ b^2 + c^2 = k^2
+
+/--
+Is there a perfect Euler brick with integer diagonal?
+-/
+@[category research open, AMS 11]
+theorem perfect_euler_brick_existence :
+    (∃ a b c d : ℕ+, IsEulerBrick a b c ∧ a^2 + b^2 + c^2 = d^2) ↔ answer(sorry) := by
```

Comment:
According to the Wikipedia these are called perfect cuboids, I think it would be nice to add that as a separate definition!